Tribler/Test/Core/test_sqlitecachedbhandler.py

```diff
@@ -59,7 +59,6 @@ def setUp(self):
         self.sqlitedb.initialize()
         self.session.sqlite_db = self.sqlitedb
 
-    @blocking_call_on_reactor_thread
     def tearDown(self):
         self.sqlitedb.close()
```

Comment:
Are all these threadsafe? (DB operations are done on the Twisted thread usually)